packages/shared/styles/helpers/_colors.scss

```diff
@@ -0,0 +1,21 @@
+@each $color, $value in $colors {
+  // Atomic utility classes for text and bg colors
+  .text-#{$color} {
+    color: $value !important;
+  }
+  .bg-#{$color} {
+    background-color: $value !important;
+  }
+
+  // General color modifiers
+  // May be overwritten by specific component modifiers
+  .is-#{$color} {
```

Comment:
maybe `color-black` or `c-black` convention is more descriptive?

Reply:
not sure if people would associate it with CSS color property (text color)...

Reply:
in css `color` is refering to text color. I usually see 'is' as a boolean state (eg `is-active`, `is-open`) @mayashavin , @przspa wdyt?

Reply:
> in css `color` is refering to text color

Yup, and that's the problem. In this case `is-primary` is not applying primary to text color, it's applying primary to bg and border color (darken), and yiq (black or white) to text color...
Maybe `theme-primary`?

Reply:
@filrak `is-active` is more as a state than a css class. @leomp12 can we just keep the standard as `color-` ? `theme` should be `dark` or `light` or `primary` (probably) and not about color specifically.

Reply:
Hey @mayashavin , I do not know if I understand you here.
I used `theme-primary` as an example but it will also be available for the other theme colors, so `theme-dark`, `theme-warning`, `theme-secondary`...

These classes will change the element bg color to respective theme color, and text color using YIQ. The point is what the prefix for these class names, `is-`, `color-`, `theme-`, `any-`.
Did you understand this way?
If yes, I'll use `color-` as suggested by @filrak and you :)

Reply:
I think you could go with `color`. We can also specify `theme-dark` option later to be applied to root component for the dark mode. I think that would be a nice addition but requires some work so maybe in next releases, not for beta.